portfolio/src/main/webapp/script.js

```diff
@@ -77,20 +77,19 @@ async function displayCommentsToPage() {
     arrayTextElement.innerHTML = 'Be the first to comment below!';
   }
 
-  for (let i = 0; i < textArray.length; i++) {
+  let i = 0;
```

Comment:
Ah, y'know what, I actually like it better the way you had it before. That's my bad, sorry